src/plugins/intel_cpu/src/mkldnn_exec_network.cpp

```diff
@@ -503,10 +505,12 @@ bool MKLDNNExecNetwork::CanProcessDynBatch(const InferenceEngine::CNNNetwork &ne
         auto type = TypeFromName(op->get_type_name());
         if (type == Tile) {
             const auto tile = std::dynamic_pointer_cast<const ngraph::opset1::Tile>(op);
+            if (!tile)
+                return false;
```

Comment:
The previous logic called `continue` if the `tile` is null, which probably means that only `opset1::Tile` node supported, but here we interrupt the whole loop. Should not we align the behavior with the previous logic? BTW `get_input_node_shared_ptr` can be called to the base node type.